CLOUDFRONT.md

```diff
@@ -112,15 +112,17 @@ cat public_key.pem
 7. _Allowed HTTP methods_: Choose _GET, HEAD, OPTIONS_ and tick _OPTIONS_ under _Cache HTTP methods_.
 8. _Restrict viewer access_: Choose _Yes -> Trusted key groups (recommended)_.
 9. Add key group created earlier
-10. _Cache key and origin requests_: Choose _Cache policy and origin request policy (recommended)_.
-11. _Cache policy_: Choose CachingOptimized
-12. _Origin request policy_: Choose CORS-S3Origin
-13. _Response headers policy_: Choose CORS-with-preflight-and-SecurityHeadersPolicy-ReadOnly
-14. Click _Create distribution_.
-15. Navigate to https://console.aws.amazon.com/cloudfront/v3/home#/distributions.
-16. Confirm, that _Status_ is _Enabled_ and _Last modified_ is changed from _Deploying_ to the date the distribution was created.
-17. Open your distribution.
-18. Write down _Distribution domain name_ (with https://).
+10. _Cache key and origin requests_: Choose _Legacy cache settings_.
```

Comment:
Some of these worry me that we've missed steps somewhere else and so changing these settings is a workaround to getting it work again. I'm not across the detail so can't know for sure but I think this should be reviewed by others in the au inf team